.github/workflows/integration-test.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     - name: Use Node.js
       uses: actions/setup-node@v1
       with:
-        node-version: '10.x'
+        node-version: '12.x'
```

Comment:
prism updated to use worker threads which was added in 10.5. github actions uses 10.2